lib/ansible/modules/monitoring/zabbix/zabbix_hostmacro.py

```diff
@@ -214,7 +213,7 @@ def main():
             # update host macro
             host_macro_class_obj.update_host_macro(host_macro_obj, macro_name, macro_value)
         else:
-            module.exit_json(changed=False, result="Host macro %s already exists and force set to no" % macro_name)
+            module.exit_json(changed=False, result="Host macro %s already exists and force set is to no" % macro_name)
```

Comment:
"is" goes one word to the left, between "force" and "set" :-)

Reply:
Thanks, this is why I shouldn't try and push stuff when I should be going home. 

Will pick up anything further tomorrow

Reply:
No problem. It was pure chance I was here at this time in any case. Have a good evening!